content/4_Gold/Shortest_Paths.mdx

```diff
@@ -78,142 +78,172 @@ Floyd-Warshall is suitable because of the low $N (N \le 500)$, and the inclusion
 **Time Complexity:** $\mathcal{O}(N^3)$
 
 <LanguageSection>
-
 <CPPSection>
 
 ```cpp
-// CodeSnip{CPP Short Template}
+#include <iostream>
+#include <vector>
+
+using std::cout;
+using std::endl;
+using std::vector;
 
-ll INF = 1e18;
+constexpr long long BIG = 1e18;  // We can't use LLONG_MAX because of overflow
 
 int main() {
-	ios::sync_with_stdio(0);
-	cin.tie(0);
 	int n, m, q;
-	cin >> n >> m >> q;
-
-	vector<vector<ll>> mat(n, vector<ll>(n, INF));
+	std::cin >> n >> m >> q;
 
+	vector<vector<long long>> min_dist(n, vector<long long>(n, BIG));
 	for (int i = 0; i < m; i++) {
-		int u, v, w;
-		cin >> u >> v >> w, u--, v--;
-		if (w < mat[u][v]) mat[u][v] = mat[v][u] = w;
+		int a, b;
```

Comment:
?? why a b c?

Reply:
that’s what the problem uses

Reply:
doesnt mean its good practice

Reply:
how come you’re fine with n and m but not this 💀

Reply:
find one person who unironically uses $a$ and $b$ for edges, and $c$ for weights

Reply:
cases does apparently

Reply:
it's fine as long as it's descriptive or it's from the problem